clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -1193,6 +1193,16 @@ static void addFortranMain(const ToolChain &TC, const ArgList &Args,
     return;
   }
 
+  const Driver &D = TC.getDriver();
```

Comment:
Move this whole block below the `// 2. GNU and similar`? Otherwise it seems that this is still part of MSVC processing.